shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -455,6 +472,11 @@ - (void)testInternalPluginsInvokeInViewDidLoad {
   FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:mockEngine
                                                                                 nibName:nil
                                                                                  bundle:nil];
+  UIView* view = viewController.view;
+  // The implementation in viewDidLoad requires the viewControllers.viewLoaded is true.
+  // Accessing the view to make sure the view loads in the memory,
+  // which makes viewControllers.viewLoaded true.
+  XCTAssertNotNil(view);
```

Comment:
This is an existing bug where it crashes in ios_debug_sim_unopt because view is not loaded into memory. I'm not sure how it passed on CI but it constantly fails on my local.